variables.tf

```diff
@@ -41,32 +41,37 @@ variable "task_container_port" {
 }
 
 variable "task_container_protocol" {
-  description = "Optional: Protocol that the container exposes."
+  description = "Protocol that the container exposes."
   default     = "HTTP"
 }
 
 variable "task_definition_cpu" {
-  description = "Optional: Amount of CPU to reserve for the task."
+  description = "Amount of CPU to reserve for the task."
   default     = "256"
 }
 
 variable "task_definition_memory" {
-  description = "Optional: Amount of memory to reserve for the task."
+  description = "The soft limit (in MiB) of memory to reserve for the container."
   default     = "512"
 }
 
+variable "task_container_command" {
```

Comment:
Same here. If we want to use `task_container_` instead of `task_definition_` we have to remember to rename the variables in the ECS module as well

Reply:
Agreed, same as previous comment, its actually defined in the container definition and not in the task definition.